src/helpers/signatures/eddsaSigPoseidon.ts

```diff
@@ -0,0 +1,18 @@
+import { BigNumber, utils } from 'ethers'
+import { buildEddsa, buildPoseidon } from 'circomlibjs'
+import env from '@/helpers/env'
+
+const privateKey = utils.arrayify(env.EDDSA_PRIVATE_KEY)
+
+export default async function (message: (number | BigNumber)[] | Uint8Array) {
+  // Message
+  const poseidon = await buildPoseidon()
```

Comment:
Should we actually build those each time? Maybe we can cache those guys into `const`?